shared/containers/AllLayout.js

```diff
@@ -0,0 +1,20 @@
+import MapLayout from './mapLayout';
```

Comment:
not 100% sure, but presenting role is component's role.
and container is for wrapping state and function property for its component.
since this component has no state, this doesn't need to be container.
Any other idea is welcomed cause i'm not 100% sure. And it's not a big problem now.